tests/actions.test.ts

```diff
@@ -1,10 +1,14 @@
+// Copyright (C) 2024, Nuklai. All rights reserved.
+// See the file LICENSE for licensing terms.
+
 import { afterAll, beforeAll, describe, expect, it } from '@jest/globals'
 import { VMABI } from 'hypersdk-client/dist/Marshaler'
 import { Block } from 'hypersdk-client/dist/apiTransformers'
 import { TransactionResult } from '../src/client'
-import { NuklaiSDK } from '../src/sdk'
+import { NuklaiSDK } from '../src'
+import {MAINNET_PUBLIC_API_BASE_URL} from "../src/endpoints";
 
-const API_HOST = 'http://127.0.0.1:9650'
```

Comment:
while this change will work for now(because i have deployed our new devnet with the address `00c4cb545f748a28770042f893784ce85b107389004d6a0e0d6d7518eeae1292d9` having all the NAI tokens, this won't be the case when we go to production. so perhaps, we don't wanna have our tests rely on the deployed devnet but rather something local running at http://127.0.0.1:9650